### PR TITLE
Public static class fields supported on Firefox 75

### PR DESCRIPTION
**Pull request by afmenez**
Public static class fields will be supported on Firefox 75:
https://wiki.developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/75#JavaScript

### CODE REVIEW

File: data-esnext.js [hunk 1]
```diff
@@ -986,7 +987,8 @@ exports.tests = [
       */},
       res: {
         firefox2: false,
-        firefox75: firefox.developer,
+        firefox74:false,
```
Comment (ljharb) [inline data-esnext.js]:
```suggestion
        firefox74: false,
```

File: data-esnext.js [hunk 2]
```diff
@@ -946,7 +946,8 @@ exports.tests = [
         tr: true,
         typescript1corejs2: true,
         firefox2: false,
-        firefox75: firefox.developer,
+        firefox74:false,
```
Comment (ljharb) [inline data-esnext.js]:
```suggestion
        firefox74: false,
```

**Review by ljharb — CHANGES_REQUESTED** [review]
Seems like this shouldn’t merge until April 7th?

**Comment by afmenez** [comment]
Why the wait?

**Comment by ljharb** [comment]
In case it doesn’t end up being released on that date. We generally don’t include things in advance of their being released, i think?

**Comment by afmenez** [comment]
Yes, we do.  We have the "unstable" tab for that.  There are 2 unstable
versions of Firefox.  And Chrome.  And Safari.

**Review by ljharb — APPROVED** [review]

**Review by ljharb — APPROVED** [review]

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**